core/src/web/clipboard.ts

```diff
@@ -22,10 +23,24 @@ export class ClipboardPluginWeb extends WebPlugin implements ClipboardPlugin {
       return Promise.reject('Clipboard API not available in this browser');
     }
 
-    if (options.string !== undefined || options.url) {
```

Comment:
@jcesarmobile Have you changed this line on purpose? This might have reintroduced issue #2366

Reply:
Oops, sorry, will fix it tomorrow as I’m working in other clipboard changes

Reply:
should be fixed again by https://github.com/ionic-team/capacitor/pull/2527